colorbleed/maya/lib.py

```diff
@@ -796,48 +796,54 @@ def assign_look_by_version(nodes, version_id):
 
     Args:
         nodes(list): nodes to assign look to
-        version_id (bson.ObjectId)
+        version_id (bson.ObjectId): database id of the version
 
     Returns:
         None
     """
     # TODO: make `iter_loader` available in as a more global function
     from avalon.tools.cbloader import lib as loaderlib
 
-    # get representations of shader file and relationships
+    # Get representations of shader file and relationships
     look_representation = io.find_one({"type": "representation",
                                        "parent": version_id,
                                        "name": "ma"})
 
-    connection_represenations = io.find_one({"type": "representation",
-                                             "parent": version_id,
-                                             "name": "json"})
-
-    # Load file
-    shader_filepath = api.get_representation_path(look_representation)
-    shader_relation = api.get_representation_path(connection_represenations)
-
-    reference_node = get_reference_node(shader_filepath)
-    if reference_node is None:
-        log.info("Loading lookdev for the first time..")
+    connection_representions = io.find_one({"type": "representation",
```

Comment:
`connection_representions` typo? :) And maybe just calling it `json` representation makes more sense too?

How about `json_representation`?